### DISCUSSION TITLE
Am I missing something in this PR?

### DISCUSSION THREAD

**Discussion by 64J0** [github-linguist/linguist#6701] [Q&A] [answered]
Am I missing something in this PR? 

- https://github.com/github-linguist/linguist/pull/6686

**Comment by lildude** [comment, marked as answer]
Yes, community buy-in... your referenced discussion only has 23 votes which is incredibly low and hardly a good indicator of over-all community agreement. This is probably because you've posted it on the discussions for the VSCode integration and not a forum dedicated to the language itself. I also don't merge PRs until close to making a new release so I tend to give these PRs as much time as possible before commenting to allow more responses from the community. 

We're very careful about changing the colours of languages as people are quite sensitive about this, especially for languages with large numbers of users or that have been using the same colour for many many years... F# has been using the current colour for nearly 14 years so we definitely need to be very very very careful about this.

I recommend you start a discussion in a place where the whole language community engages, not just those interested in the VSCode integration.

PS. This should have been a question on the PR itself 😉

**Reply by 64J0** [reply]
Got it. Thanks for the clarification.

> This is probably because you've posted it on the discussions for the VSCode integration and not a forum dedicated to the language itself.

Just one correction, I did not post on VS Code-related repo, but the language repo itself.
